libcnb/src/generic.rs

```diff
@@ -22,17 +23,21 @@ impl Display for GenericError {
 
 /// A generic platform that only provides access to environment variables.
 pub struct GenericPlatform {
-    env: PlatformEnv,
+    env: Env,
+}
+
+impl GenericPlatform {
```

Comment:
This appears unused - presume here just for consistency?

Reply:
(this was meant to be on the `new()` definition)

Reply:
For eventual testing when we want to create GenericPlatform without reading files from disk. :)